devops/ansible/roles/nginx/tasks/main.yml

```diff
@@ -0,0 +1,62 @@
+---
+- name: Install Firewalld
+  yum: name=firewalld state=present
+  ignore_errors: yes
+
+- name: Firewalld service state
+  service: name=firewalld state=started enabled=yes
+
+- name: Define nginx_user.
+  set_fact:
+    nginx_user: "{{ __nginx_user }}"
+  when: nginx_user is not defined
+
+- name: Enable nginx repo.
+  template:
+    src: nginx.repo.j2
+    dest: /etc/yum.repos.d/nginx.repo
+    owner: root
+    group: root
+    mode: 0644
+  when: nginx_yum_repo_enabled
```

Comment:
This seems like an odd way to do this. Why a template rather than using Ansible yum or package module? (No need to answer here. If there is an explanation, add comment above task.)